### PR TITLE
Genkozh dev

### PR DESCRIPTION
**Pull request by DmyMi**

### CODE REVIEW

**Comment by takvol** [comment]
`.DS_Store` files and `.vscode` folder should be removed and added to .gitignore. Though it may be done in the next pull request.

File: src/web/web.ts [hunk 1]
```diff
@@ -28,8 +28,8 @@ app.get("/*", (req, res) => {
 
 export const start = (port: number): Promise<void> => {
   return new Promise<void>((resolve, reject) => {
-    const server = app.listen(process.env.PORT || 5000, function() {
-      console.log(`Listening on port ${process.env.PORT} 3000`);
+    const server = app.listen(process.env.PORT || 5001, function() {
+      console.log(`Listening on port ${process.env.PORT} 8000`);
```
Comment (takvol) [inline src/web/web.ts]: Why 5001 port?

Also ```console.log(`Listening on port ${process.env.PORT} 8000`);``` will always output 8000.
Better rewrite it to something like:
```
const port = process.env.PORT || 5001;
const server = app.listen(port, function() {
  console.log(`Listening on port ${port}`);
}
```

Comment (GenKozh) [reply src/web/web.ts]: What is process.env.PORT? Code reviewing suppose at least understanding of the code, being reviewed.

**Comment by GenKozh** [comment]
> `.DS_Store` files and `.vscode` folder should be removed and added to .gitignore. Though it may be done in the next pull request.
> 
> Also, why do we need those?
> 
> ```
> src/web/frontend/images/Frame 1.png
> src/web/frontend/images/Frame 2.png
> src/web/frontend/images/Frame 8.png
> ```

Those are stubs to simulate React routes.

File: src/web/web.ts (hunk 1 above)
Comment (GenKozh) [reply src/web/web.ts]: process.env.PORT is needed for heroku deployment. Back-end will be refactored during back-end develpoment process.

**Comment by GenKozh** [comment]
> `.DS_Store` files and `.vscode` folder should be removed and added to .gitignore. Though it may be done in the next pull request.
> 
> Also, why do we need those?
> 
> ```
> src/web/frontend/images/Frame 1.png
> src/web/frontend/images/Frame 2.png
> src/web/frontend/images/Frame 8.png
> ```

adding files to .gitignore after commit will not remove it from repository. https://stackoverflow.com/questions/18393498/gitignore-all-the-ds-store-files-in-every-folder-and-subfolder I will check in the future commits if described solution worked.

**Pull request closed without merging**